app/javascript/src/javascripts/forum_posts.js

```diff
@@ -78,14 +78,15 @@ ForumPost.vote = function(evt, score) {
 
 ForumPost.vote_remove = function(evt) {
   evt.preventDefault();
-  const id = $(evt.target.parentNode).data('forum-id');
+  const id = $(evt.currentTarget).data("forum-id");
   $.ajax({
     url: `/forum_posts/${id}/votes.json`,
-    type: 'DELETE',
-    dataType: 'json',
-    accept: 'text/javascript',
+    type: "DELETE",
+    dataType: "json",
+    accept: "text/javascript",
   }).done(function(data) {
     $(evt.target).parents(".own-forum-vote").remove();
+    $(`#forum-post-votes-for-${id} .forum-post-vote-block`).show();
     Utility.notice("Vote removed.");
```

Comment:
Perhaps, you should consider removing the notification on successful unvote?
This notification seems redundant – it should be relatively obvious already that the vote was removed.